Modules/hpe3par_remote_copy.py

```diff
@@ -353,19 +378,56 @@
       storage_system_password: password
       storage_system_username: username
       state: add_volume
-      remote_copy_group_name: farhan_rcg
+      remote_copy_group_name: test_rcg
       volume_name: demo_volume_2
       admit_volume_targets:
       - target_name: CSSOS-SSA06
         sec_volume_name: demo_volume_2
 
-  - name: Modify Remote Copy Group farhan_rcg
+  - name: admit Remote Copy target
```

Comment:
Be consistent in the name of the task. Always start with Capital letter.
Check this in other places too.